docs/admin/scale.md

```diff
@@ -123,70 +31,171 @@ Since Coder's performance is highly dependent on the templates and workflows you
 support, you may wish to use our internal scale testing utility against your own
 environments.
 
-> Note: This utility is intended for internal use only. It is not subject to any
-> compatibility guarantees, and may cause interruptions for your users. To avoid
-> potential outages and orphaned resources, we recommend running scale tests on
-> a secondary "staging" environment. Run it against a production environment at
-> your own risk.
+> Note: This utility is experimental. It is not subject to any compatibility
+> guarantees, and may cause interruptions for your users. To avoid potential
+> outages and orphaned resources, we recommend running scale tests on a
+> secondary "staging" environment or a dedicated
+> [Kubernetes playground cluster](https://github.com/coder/coder/tree/main/scaletest/templates).
```

Comment:
That template is for the scaletest runner -- if you want to spin up a k8s playground you instead want to look in the `terraform` dir.

```suggestion
> [Kubernetes playground cluster](https://github.com/coder/coder/tree/main/scaletest/terraform).
```